docs/annexes/spdx-license-expressions.md

```diff
@@ -181,79 +181,67 @@ When required to express an order of precedence that is different from the defau
 For instance, the following expression:
 
 ```text
-MIT AND (LGPL-2.1-or-later OR BSD-3-Clause)
+(LGPL-2.1-or-later OR BSD-3-Clause) AND MIT
```

Comment:
```suggestion
MIT AND (LGPL-2.1-or-later OR BSD-3-Clause)
```

I think the original author of this example purposefully put `AND` at the front, before `OR`, to demonstrate that the parentheses can be used to let `OR` applied before - despite its location at the back. So I suggest we keep the original example.

Reply:
But this reorders the terms from the previous example. The idea was to show and explain the expression without parentheses and then say "if you want to do the other way, add parentheses".